nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -112,6 +112,17 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor RIGHT_BOUND_WHERE = new PropertyDescriptor.Builder()
```

Comment:
On my side, I've tested it on MS SQL databases. I agree that people will likely enable this option in all cases and that's just adding a WHERE clause in the generated queries. I don't see any reason why this would not work / would break existing workflows. I'm fine with removing the property or setting it to true by default in case of unforeseen behaviours. What do you think?